src/App.js

```diff
@@ -31,6 +29,7 @@ import { collection, doc, getDocs } from "firebase/firestore";
 import MapComponent from "./components/map/ITEMmap";
 import { db } from "./firebase";
 import SellerForm from "./pages/SellerRegistration";
+import ScrollToTop from "./components/ScrollToTop"; // Import the ScrollToTop component
```

Comment:
Hey @kushagra-singhh 
Under the components directory, create a new folder named ScrollToTop.